tools/postman-collections/am-lib-testing.postman_collection.json

```diff
@@ -130,6 +130,34 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "Revoke Resource Access",
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"resourceId\": \"my-super-resource-id\",\n    \"accessorId\": \"UNIQUE-USER-ID\",\n    \"explicitPermissions\": [\"READ\", \"CREATE\", \"UPDATE\"],\n    \"accessType\": \"user\",\n    \"serviceName\": \"Service 1\",\n    \"resourceType\": \"Resource Type 1\",\n    \"resourceName\": \"resource\",\n    \"attribute\": \"\",\n    \"securityClassification\": \"Public\"\n}"
+				},
+				"url": {
+					"raw": "localhost:3703/lib/create-resource-access",
+					"host": [
+						"localhost"
+					],
+					"port": "3703",
+					"path": [
+						"lib",
+						"create-resource-access"
```

Comment:
Not sure what these are but revoke request should not mention create endpoint. The same in `localhost:3703/lib/create-resource-access` above.

Reply:
you're right, it just hasn't added the remove resource stuff. Odd. Good spot.